README.md

```diff
@@ -41,7 +41,7 @@ console.log(`Submitted with hash ${txHash}`);
 Make sure you add the following additional to `Settings/Developer` section:
 
 ```json
-{"SpecIndex": "u32",
+{"SpecIndex": "Vec<u8>",
```

Comment:
Should this be `String` instead of `Vec<u8>` which is technically ascii?

Reply:
We're not using `String` because it's not supported in WASM, though `Vec<u8>` can represent a string.

Reply:
Oh, don't they both map onto uint64? I was under the impression wasm only had a handful types: (un)signed int32, 64, float32, 64 (and of course memory regions).

Reply:
Regardless I believe it's common to use `Vec<u8>` in Substrate, see: https://github.com/paritytech/substrate/wiki/FAQ